index.html

```diff
@@ -13,19 +13,24 @@
 
         <nav class="nav-list">
             
-            <a class="logo" href="#">My Logo</a>
+            <a class="logo" href="#">Leslie Gudo</a>
+            <ul class="navbar-list">
 
-            <button class="menu-button">
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
```

Comment:
[OPTIONAL]I suggest you fix the mobile styling for your site. The headline section does not match the figma template. 
This is your design:
<img width="381" alt="Screenshot 2023-01-20 at 09 12 27" src="https://user-images.githubusercontent.com/48546992/213629952-b0fbf59a-7148-474e-959e-998acfe59f60.png">

this is the figma template:
<img width="381" alt="Screenshot 2023-01-19 at 09 40 33" src="https://user-images.githubusercontent.com/48546992/213630082-288af360-93cf-4ee9-b98d-5b01d898cb33.png">

Remember, that it's important to show that given a template, you can create a matching design.